Sources/CoreCommands/Options.swift

```diff
@@ -477,6 +477,12 @@ public struct BuildOptions: ParsableArguments {
     )
     public var linkTimeOptimizationMode: LinkTimeOptimizationMode?
 
+    @Flag(help: .hidden)
+    public var enableGetTaskAllowEntitlement: Bool = false
+
+    @Flag(help: .hidden)
+    public var disableGetTaskAllowEntitlement: Bool = false
```

Comment:
you can probably marge this into one settings, see other Bools

Reply:
It doesn't look `@Flag` supports optional types, unless I'm something? Maybe @natecook1000 or @rauhul can clarify?

Reply:
IIRC I had to provide a default value, but the default value depended on values of other options. I assume `false` would mean `--disable...`, `true` would mean `--enable...`, and `nil` would mean neither were passed on CLI?

Reply:
Hmm so you want to infer true/false based on other flags unless explicitly specified... that is reasonable. I don't think we support Optional Flags yet, but I wonder if you could use an EnumerableFlag instead.

Reply:
You can actually have a boolean flag as long as there's an inversion, which I think is exactly what you want in this case. The declaration would look like:

```swift
@Flag(inversion: .prefixedEnableDisable)
var getTaskAllowEntitlement: Bool?
```

It looks like we might not allow an `= nil` in there, which may have been what you ran into. That's also an issue we should be able to fix pretty easily.

Reply:
Ooh good catch!

Reply:
yes, this is what I had in mind. @MaxDesiatov

Reply:
My bad, it actually does support an option, at least with `.prefixedEnableDisable` it works great. I'll submit a follow-up PR.

Reply:
Addressed in https://github.com/apple/swift-package-manager/pull/7028